budgetApp1/budgetApp1/AccountViewModel.swift

```diff
@@ -16,31 +16,29 @@ class AccountViewModel {
  
     private var repository: AccountRepositoryType?
     private weak var delegate: AccountViewModelDelegate?
-    private var AccountList: [Account]?
+    private var accountList: [Account]?
     
     init(repository: AccountRepositoryType, delegate: AccountViewModelDelegate) {
         self.repository = repository
         self.delegate = delegate
     }
     
-    var AccountListCount: Int {
-        return AccountList?.count ?? 0
+    var accountListCount: Int {
+        return accountList?.count ?? 0
     }
     
     func account(atIndex: Int) -> Account? {
-        return AccountList?[atIndex] ?? nil
+        return accountList?[atIndex] ?? nil
```

Comment:
same here. If swift lint is not picking it up, there is a problem with it. If it is, lease don't ignore the warnings😄